pkg/cmd/roachtest/tests/restore.go

```diff
@@ -940,6 +963,26 @@ func (sp restoreSpecs) run(ctx context.Context, c cluster.Cluster) error {
 	return nil
 }
 
+// initPerfRecorder returns a function to a record perf metric for the given test
```

Comment:
I think we have a very similar method in `initBulkJobPerfArtifacts`? Any reason to keep both around that I'm missing. Also IIRC I had to add the restore test names to some file in roachperf so that they were picked up for the special bulk job rendering we do. Things might have changed since I looked at roachperf though so maybe thats not the case anymore.

Reply:
I just changed this stuff substantially. Essentially, initBulkJobArtifacts merely records the duration of the test and computes throughput in the roachperf update process by relying on parsing the test name. This is finicky and can lead to inaccurate bench data... bank2tb doesn't actually restore 2tb of physical data for example. 

I'd rather compute and record throughput in the roach test process, and remove the parsing in the roachperf process. I'll add something to the commit message which explains this.